api/draft_registrations/views.py

```diff
@@ -230,3 +230,17 @@ def get_serializer_context(self):
         context['resource'] = self.get_draft()
         context['default_email'] = 'draft'
         return context
+
+
+class DraftBibliographicContributorsList(DraftContributorsList):
+
+    view_name = 'draft-registration-bibliographic-contributor-detail'
+
+    def get_default_queryset(self):
+        # Overrides NodeContributorsList
+        draft = self.get_draft()
+        return draft.draftregistrationcontributor_set.filter(visible=True).include('user__guids')
+
+    # Read-only
+    def get_serializer_class(self):
```

Comment:
Nit: Any reason you overrode get_serializer_class instead of just setting `serializer_class`

Reply:
Yeah my comment was crap, but it's read-only because it's overriding the serializer class to not be conditional on HTTP method. So no Create serializer.